USERPREFS.md

```diff
@@ -69,6 +69,11 @@ Changes should be first tested on user-prefs-master to be later applied to
 hasura migrate apply --endpoint https://user-prefs-development.herokuapp.com
 hasura metadata apply --endpoint https://user-prefs-development.herokuapp.com
 ```
+If you update a table that is used as a GraphQL Enum, such as `lucumaGridLayoutPositions`,
```

Comment:
Thanks this is good, I have done this sometimes manually but didnn't include it in the docs